models/mongodb/status.js

```diff
@@ -5,18 +5,16 @@ var Schema = mongoose.Schema
 
 var Status = new Schema({
     penalties: [{
-        type: String,
-        index: true
+        type: String
     }],
     masternodes: [{
-        type: String,
-        index: true
+        type: String
     }],
     proposes: [{
-        type: String,
-        index: true
+        type: String
     }],
-    epoch: { type: Number, index: true }
+    epoch: { type: Number, index: true },
+    created_at: { type: Date }
```

Comment:
use `createdAt` instead of `created_at`

Reply:
The field `created_at` is used to store timestamp from a block checkpoint. When creating a new record I cannot set `createdAt` field to timestamp(it always take current time), so I have to `created_at` field to store timestamp.

Reply:
use `timestamp` or `blockCreatedAt`. just concert about coding convention.